packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -166,10 +166,12 @@ export class AuthenticatorService implements OnDestroy {
     };
   }
 
-  /** @deprecated For internal use only */
-  public subscribe(callback: AuthSubscriptionCallback) {
+  public subscribe(callback: AuthSubscriptionCallback): () => void {
     if (this._authService) {
-      return this._authService.subscribe(callback);
```

Comment:
For context, this internal function was just directly using `xstate` subscription instead of facades.